travis.yml

```diff
@@ -0,0 +1,8 @@
+language: java
```

Comment:
seems like the name of this file is wrong

and let's see if something should be added for it to work with spring boot